src/pages/settings/client-portal/components/Settings.tsx

```diff
@@ -53,7 +57,7 @@ export function Settings() {
 
   return (
     <Card title={t('settings')}>
-      {isHosted() && (
+      {isHosted() && isCompanySettingsActive && (
```

Comment:
@beganovich Wherever you encounter this type of condition, it indicates that the field/fields are available in company settings but not in group_settings.